Misha/OB-365_erat_sieve/utest_sieve_of_erat.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from main import sieve_of_eratosthenes
+
+
+class EratSieveTest(unittest.TestCase):
```

Comment:
Add two more tests
INput:
199
and
100

Reply:
added

Reply:
it works correctly, its the same test as with 5 and 6 values